CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 #### Features
 
 * Your contribution here.
+* [#1658](https://github.com/ruby-grape/grape/pull/1658): Update `rack-test` to `0.7.0` and fix failing specs - [@ashkan18](https://github.com/ashkan18).
```

Comment:
I would remove "and fix failing specs" cause that's not very interesting or useful.